easybuild/easyblocks/a/amber.py

```diff
@@ -157,11 +164,15 @@ def install_step(self):
         else:
             raise EasyBuildError("Don't know how to compile with compiler family '%s' -- check EasyBlock?", comp_fam)
 
+        # The NAB compiles need openmp flag
+        if self.toolchain.options.get('openmp', None):
+            env.setvar('CUSTOMBUILDFLAGS', self.toolchain.get_flag('openmp'))
```

Comment:
Hmm, shouldn't we pass the full `$CFLAGS` here? If OpenMP is enabled via `toolchainopts`, it will include `-fopenmp`.

Reply:
Amber doesn't quite use EB's CFLAGS at all, yet. So at the moment adding just the openmp flag is what should be done for consistency until that is dealt with.

Reply:
Well, isn't this an opportunity to do so? What would be the correct way to let Amber honor the compiler flags put in place by EasyBuild? Wouldn't it be defining `$CUSTOMBUILDFLAGS` equal to `$CFLAGS`?

Reply:
Fixing that requires more patching of makefiles that i haven't had time to dive into. The amber build is somewhat convoluted... There are places where CUSTOMBUILDFLAGS isn't used for instance.
Getting this right is not trivial.